diff --git a/src/Online/Online.cpp b/src/Online/Online.cpp
index 28a2230..54d1842 100644
--- a/src/Online/Online.cpp
+++ b/src/Online/Online.cpp
@@ -49,7 +49,7 @@ void online_phase(int online_num, Player &P, offline_control_data &OCD,
       OTD.aBD.aBD_mutex.lock();
       if (OTD.ready == false)
         {
-          wait= true;
+          //wait= true;
         }
       OTD.aBD.aBD_mutex.unlock();
 
diff --git a/src/System/RunTime.cpp b/src/System/RunTime.cpp
index 43eb6b4..f0f0808 100644
--- a/src/System/RunTime.cpp
+++ b/src/System/RunTime.cpp
@@ -122,7 +122,7 @@ void Run_Scale(unsigned int my_number, unsigned int no_online_threads,
   // Add in the FHE threads
   unsigned int tnthreads= nthreads + number_FHE_threads;
   // Add in the OT threads
-  tnthreads+= 2;
+//  tnthreads+= 2;
   daBitMachine.Initialize(SD.n, OCD);
 
   /* Initialize the networking TCP sockets */
